internal/providers/pluginfw/resources/qualitymonitor/resource_quality_monitor_test.go

```diff
@@ -151,3 +153,43 @@ func TestUcAccUpdateQualityMonitorPluginFramework(t *testing.T) {
 		`,
 	})
 }
+
+func TestUcAccQualityMonitorImportPluginFramework(t *testing.T) {
+	if os.Getenv("GOOGLE_CREDENTIALS") != "" {
+		t.Skipf("databricks_quality_monitor resource is not available on GCP")
+	}
+	acceptance.UnityWorkspaceLevel(t, acceptance.Step{
+		Template: commonPartQualityMonitoring + `
+
+			resource "databricks_quality_monitor_pluginframework" "testMonitorInference" {
+				table_name = databricks_sql_table.myInferenceTable.id
+				assets_dir = "/Shared/provider-test/databricks_quality_monitoring/${databricks_sql_table.myInferenceTable.name}"
+				output_schema_name = databricks_schema.things.id
+				inference_log = {
+				  granularities = ["1 day"]
+				  timestamp_col = "timestamp"
+				  prediction_col = "prediction"
+				  model_id_col = "model_id"
+				  problem_type = "PROBLEM_TYPE_REGRESSION"
+				} 
+			}
+
+			output "table_name" {
+				value = databricks_quality_monitor_pluginframework.testMonitorInference.table_name
+			}
+		`,
+	},
+		acceptance.Step{
+			ImportState:  true,
+			ResourceName: "databricks_quality_monitor_pluginframework.testMonitorInference",
+			ImportStateIdFunc: func(s *terraform.State) (string, error) {
+				// Fetch the output value of 'table_name' from the state
+				rs, ok := s.RootModule().Outputs["table_name"]
+				if !ok || rs.Value == "" {
+					return "", fmt.Errorf("table_name output not found in the first step")
+				}
+				return rs.Value.(string), nil
+			},
```

Comment:
Can we set `ImportStateVerify` to `true` here? I believe this will check that import resulted in the same state as the resource we created above. If we do, we may need to set `ImportStateVerifyIdentifierAttribute` to `table_name` as that is the "id" of the resource.